modules/imaging_browser/php/mrinavigation.class.inc

```diff
@@ -27,6 +27,9 @@ namespace LORIS\imaging_browser;
 
 class MRINavigation
 {
+
+    var $urlParams = array();
```

Comment:
```suggestion
    public $urlParams = array();
```

Since this is new and we don't need to support ancient versions of PHP, we should be using visibility modifiers instead of "var" (I didn't look into if this should be more strict than public, but the suggested change suggests public since that's equivalent to var..)